checker/tests/initialization/EISOPIssue363b.java

```diff
@@ -0,0 +1,63 @@
+// Test case for issue 363:
+// https://github.com/eisop/checker-framework/issues/363
+
+import org.checkerframework.checker.initialization.qual.Initialized;
+import org.checkerframework.checker.initialization.qual.NotOnlyInitialized;
+import org.checkerframework.checker.initialization.qual.UnknownInitialization;
+
+class MyException1 extends Exception {
```

Comment:
What is the difference between this file and the other file? Can you add a comment at the beginning of both test files to explain how you split up the tests?

Reply:
> What is the difference between this file and the other file? Can you add a comment at the beginning of both test files to explain how you split up the tests?

This file is for multiple throw clauses in the method signature. Do you think it is better keep them in a separate file?